WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductTypeBottomSheetListSelectorCommand.swift

```diff
@@ -0,0 +1,102 @@
+import Yosemite
+
+private extension ProductType {
+    /// Title shown on the action sheet.
+    ///
+    var actionSheetTitle: String {
+        switch self {
+        case .simple:
+            return NSLocalizedString("Simple product",
+                                     comment: "Action sheet option Simple when the user wants to change the Product type")
+        case .variable:
+            return NSLocalizedString("Variable product",
+                                     comment: "Action sheet option Variable when the user wants to change the Product type")
+        case .grouped:
+            return NSLocalizedString("Grouped product",
+                                     comment: "Action sheet option Grouped when the user wants to change the Product type")
+        case .affiliate:
+            return NSLocalizedString("External product",
+                                     comment: "Action sheet option External when the user wants to change the Product type")
+        case .custom(let title):
+            return title
+        }
+    }
+
+    /// Description shown on the action sheet.
+    ///
+    var actionSheetDescription: String {
+        switch self {
+        case .simple:
+            return NSLocalizedString("A unique item to sell",
+                                     comment: "Description of the Action sheet option  for the product type Simple")
```

Comment:
nit: "Description of the Action sheet option when the user wants to change the Product type to simple product" (and similar for other options)?

Reply:
👍 done here 2fe635cf92b2e62ba0f4310a39826176cc598c65